src/widget/style.cpp

```diff
@@ -228,7 +229,7 @@ QFont Style::getFont(Style::Font font)
         appFont(defSize - 1, QFont::Light),  // small light
     };
 
-    return fonts[font];
+    return fonts[static_cast<int>(font)];
```

Comment:
Unrelated to this PR, but `fonts` should probably be a map, not an array? :x Would make this less spooky. Especially since there's nothing around the Font enum to indicate that it can't be reordered or anything. :P

Reply:
Yeah agreed, moved it to a std::map.